lib/cw-core/src/util.rs

```diff
@@ -57,3 +57,323 @@ pub fn split_framebuffer(framebuffer: &[u8]) -> Vec<Vec<u8>> {
 
     framebuffers
 }
+
+pub fn get_different_framebuffer(previous_framebuffer: &mut [u8], framebuffer: &[u8]) -> Vec<u8> {
```

Comment:
This name is really accurate for what is does, I would change it to something like:

1. `extract_changed_values_framebuffer`
2. `extract_,mutated_pixels`

But something else is also fine


Also if you would be able to write a documentation comment for this function, that would really help with understanding this code